Makefile

```diff
@@ -255,11 +255,26 @@ ifneq ($(ADD_LDFLAGS), NONE)
 endif
 
 ifeq ($(NVCC), NONE)
+	# If NVCC has not been manually defined, use the CUDA_PATH bin dir.
 	ifneq ($(USE_CUDA_PATH), NONE)
 		NVCC=$(USE_CUDA_PATH)/bin/nvcc
 	endif
 endif
 
+# Gaurd against displaying nvcc info messages to users not using CUDA.
+ifeq ($(USE_CUDA), 1)
+	# If NVCC is not at the location specified, use CUDA_PATH instead.
+	ifeq ("$(wildcard $(NVCC))","")
```

Comment:
Is there a difference between empty and none?

Reply:
Just to clarify, this resolves the filepath and is basically a ```file.exists```, right?

Reply:
I'm surely not a Makefile expert, but I assume None is getting picked up from the config files, i.e. here: https://github.com/apache/incubator-mxnet/blob/master/make/config.mk#L69

Yes this is file exists, testable by editing the config's nvcc to point to various exiting and non-existing paths, then observing logging messages.  Documented quite well here: http://www.humbug.in/2012/makefile-check-if-a-file-exists-using-wildcard-function/